guides/source/active_record_multiple_databases.md

```diff
@@ -222,13 +222,10 @@ class Dog < AnimalsRecord
 end
 ```
 
-Note: Since Rails doesn't know which database is the replica for your writer you will need to
-add this to the abstract class after you're done.
-
```

Comment:
Sorry, but I noticed that this is not fully addressed at the previous document.
So, I will revert this removal and squash the commit. (Maybe we can find the best place afterwards)